app/models/partnership.rb

```diff
@@ -28,4 +30,12 @@ def challenge!(reason)
 
     update!(challenge_reason: reason, challenged_at: Time.zone.now)
   end
+
+  def challenge_deadline
+    created_at + CHALLENGE_WINDOW
+  end
```

Comment:
I think we should store challenge_deadline in the separate column - there is a business question of "What do we want to happen to already existing Partnerships when the decision is made to shorten or extend the CHALLENGE_WINDOW".  In general I try avoiding any business logic depending on `created_at` column - it feels more like a database-level metadata than the domain data. Common issue would be importing pre-existing data - we would like to know both information - when the record was created (db-metadata) and when the partnership started.